scripts/test/shared.py

```diff
@@ -184,7 +184,9 @@ def is_exe(fpath):
 WASM_METADCE = [os.path.join(options.binaryen_bin, 'wasm-metadce')]
 WASM_EMSCRIPTEN_FINALIZE = [os.path.join(options.binaryen_bin,
                                          'wasm-emscripten-finalize')]
-BINARYEN_JS = os.path.join(options.binaryen_root, 'out', 'binaryen.js')
+# Due to cmake limitations, we emit binaryen_js.js (see CMakeLists.txt
+# for why).
+BINARYEN_JS = os.path.join(options.binaryen_root, 'bin', 'binaryen_js.js')
```

Comment:
options.binaryen_bin like above?  Or does it not get installed?

Reply:
I think it does, thanks, fixing.